pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/BrokersImpl.java

```diff
@@ -352,9 +353,10 @@ public void failed(Throwable throwable) {
     }
 
     @Override
+    @Deprecated
     public void healthcheck() throws PulsarAdminException {
         try {
-            healthcheckAsync().get(this.readTimeoutMs, TimeUnit.MILLISECONDS);
+            healthcheckAsync("V1").get(this.readTimeoutMs, TimeUnit.MILLISECONDS);
```

Comment:
query param is optional and if param is null then it should consider V1 by default. so, we should not change the behavior and let it be empty which can validate that the existing behavior is working.

Reply:
done